doc/usage/installation.rst

```diff
@@ -12,9 +12,13 @@ Installing Sphinx
 Overview
 --------
 
-Sphinx is written in `Python`__ and supports Python 3.6+.
+Sphinx is written in `Python`__ and supports Python 3.6+. It builds upon the
+shoulders of many third-party libraries such as `Docutils`__ and `Jinja`__,
+which are installed when Sphinx is installed.
 
 __ https://docs.python-guide.org/
+__ http://docutils.sourceforge.net/
```

Comment:
It seems my browser is redirected into https://docutils.sourceforge.io/.

Reply:
Done